armi/cli/modify.py

```diff
@@ -61,6 +61,13 @@ def addOptions(self):
             default=".",
             help="A root directory in which to search for settings files, e.g., armi/tests.",
         )
+        self.parser.add_argument(
+            "--style",
```

Comment:
I'm wondering if we want a more descriptive name for this option? "style" could refer to a lot of things, and maybe we don't want to occupy such a generic name.

Maybe something like "--settingsDumpStyle"?